pkg/controller/operators/olm/config.go

```diff
@@ -33,6 +33,7 @@ type operatorConfig struct {
 	apiLabeler        labeler.Labeler
 	restConfig        *rest.Config
 	configClient      configv1client.Interface
+	copiedCSVs        bool
```

Comment:
suggestion: its sometimes helpful to make boolean variables clear that they are boolean in their naming. E.g. by adding the `is` prefix. Wdyt of, instead of `copiedCSVs` we call it `isCopiedCSVsEnabled` or something like that? `copiedCSVs` reads a bit like it should be a slice.